src/main/java/io/stargate/sgv2/jsonapi/exception/mappers/ThrowableToErrorMapper.java

```diff
@@ -70,7 +71,12 @@ public final class ThrowableToErrorMapper {
           if (message.contains("vector<float,")) { // TODO is there a better way?
             message = "Mismatched vector dimension";
           }
+          fields = Map.of("errorCode", ErrorCode.INVALID_REQUST.name());
           return new CommandResult.Error(message, fieldsForMetricsTag, fields, Response.Status.OK);
+        } else if (throwable instanceof DriverTimeoutException
+            || throwable instanceof WriteTimeoutException) {
```

Comment:
Should we add `ReadTimeoutException` also here?

Reply:
Added, as part of the PR I just moved it above Driver exception, since *TimeoutException are also DriverException

Reply:
Thanks.
Yes, I saw that you just moved it. Looking closer, I thought it might be correct to add ReadTimeoutException also.